src/applications/personalization/profile/components/personal-information/PersonalInformationSection.jsx

```diff
@@ -10,17 +10,35 @@ import { renderDOB } from '@@profile/util/personal-information/personalInformati
 import { CONTACTS } from '@department-of-veterans-affairs/component-library/Telephone';
 import ProfileInfoTable from '../ProfileInfoTable';
 import { ProfileInfoCard } from '../ProfileInfoCard';
-import GenderIdentityAdditionalInfo from './GenderIdentityAdditionalInfo';
+import GenderIdentityDescription from './GenderIdentityDescription';
 import LegalName from './LegalName';
 import DisabilityRating from './DisabilityRating';
 import { Toggler } from '~/platform/utilities/feature-toggles';
 
+const LegalNameDescription = () => (
+  <va-additional-info trigger="How to update your legal name">
+    <p className="vads-u-margin-top--0">
+      If you’ve changed your legal name, you’ll need to tell us so we can change
+      your name in our records.
+    </p>
+    <p className="vads-u-margin-bottom--0">
+      <a href="/resources/how-to-change-your-legal-name-on-file-with-va">
+        Learn how to change your legal name on file with VA
+      </a>
+    </p>
+  </va-additional-info>
+);
+
 const PersonalInformationSection = ({
   dob,
   shouldShowPronounsAndSexualOrientation,
 }) => {
   const tableFields = [
```

Comment:
not launch-blocking: when the [PR to remove ProfileInfoTable](https://github.com/department-of-veterans-affairs/vets-website/pull/25137#pullrequestreview-1567529118) is merged, will you rename this to cardFields?